File: src/dotnet/ArdsTools/Ards.NetCore.Adif.Api/Models/Continent.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Ards.NetCore.Adif.Api.Models
{
    [Table("continent", Schema = "adif")]
    public partial class Continent
    {
        [Column("id")]
        public int Id { get; set; }
        [Column("abbreviation")]
        [StringLength(2)]
        public string Abbreviation { get; set; }
        [Column("name")]
        [StringLength(14)]
        public string Name { get; set; }
    }
}

File: src/java/ArdsAdifApi/src/main/java/com/ards/adif/api/model/Continent.java
package com.ards.adif.api.model;

import java.io.Serializable;
import javax.persistence.*;


/**
 * The persistent class for the continent database table.
 * 
 */
@Entity
@Table(name="continent")
@NamedQuery(name="Continent.findAll", query="SELECT c FROM Continent c")
public class Continent implements Serializable {
	private static final long serialVersionUID = 1L;

	@Id
	@Column(unique=true, nullable=false)
	private Integer id;

	@Column(length=2)
	private String abbreviation;

	@Column(length=14)
	private String name;

	public Continent() {
	}

	public Integer getId() {
		return this.id;
	}

	public void setId(Integer id) {
		this.id = id;
	}

	public String getAbbreviation() {
		return this.abbreviation;
	}

	public void setAbbreviation(String abbreviation) {
		this.abbreviation = abbreviation;
	}

	public String getName() {
		return this.name;
	}

	public void setName(String name) {
		this.name = name;
	}

}